untitled/src/main/java/Person.java

```diff
@@ -0,0 +1,11 @@
+public class Person {
+    private final String name;
+
+    public Person(String name) {
+        this.name = name;
+    }
```

Comment:
인자의 이름은 input이 더 적절한것같아요.

```suggestion
    public Person(String input) {
        this.name = input;
    }
```